ops/model.py

```diff
@@ -1453,13 +1465,21 @@ class Relation:
     This is accessed using, for example, ``Relation.data[unit]['foo']``.
     """
 
+    active: bool
+    """Indicates whether this relation is active.
+
+    This will be ``False`` if the current event is a ``relation-broken`` event
+    associated with this relation.
```

Comment:
I think it'd be good to mention the (normal) True case. Perhaps:

```suggestion
    This is normally ``True``; it will be ``False`` if the current event is a
    ``relation-broken`` event associated with this relation.
```